bin/run-in-docker.sh

```diff
@@ -1,39 +1,71 @@
 #!/usr/bin/env bash
 set -e
 
-# Synopsis:
-# Test runner for run.sh in a docker container
-# Takes the same arguments as run.sh (EXCEPT THAT SOLUTION AND OUTPUT PATH ARE RELATIVE)
-# Builds the Dockerfile
-# Runs the docker image passing along the initial arguments
+GREEN="\033[0;32m"
+ORANGE="\033[0;33m"
+NC="\033[0m"
 
-# Arguments:
-# $1: exercise slug
-# $2: **RELATIVE** path to solution folder (with trailing slash)
-# $3: **RELATIVE** path to output directory (with trailing slash)
-
-# Output:
-# Writes the test results to a results.json file in the passed-in output directory.
-# The test results are formatted according to the specifications at https://github.com/exercism/automated-tests/blob/master/docs/interface.md
-
-# Example:
-# ./run-in-docker.sh two-fer ./relative/path/to/two-fer/solution/folder/ ./relative/path/to/output/directory/
-
-# If arguments not provided, print usage and exit
-if [ -z "$1" ] || [ -z "$2" ] || [ -z "$3" ]; then
-    echo "usage: run-in-docker.sh exercise-slug ./relative/path/to/solution/folder/ ./relative/path/to/output/directory/"
+printHelpAndExit() {
+    echo -e "${ORANGE}USAGE${NC}"
+    echo -e "   ./run-in-docker.sh ${GREEN}-s${NC} two-fer ${GREEN}-i${NC} /absolute/path/to/solution/folder/ ${GREEN}-o${NC} ./absolute/path/to/output/directory/"
+    echo
+    echo -e "${ORANGE}SYNOPSIS${NC}"
+    echo "   Test runner for run.sh in a docker container;"
+    echo "   Takes the same arguments as run.sh (EXCEPT THAT SOLUTION AND OUTPUT PATH ARE RELATIVE);"
+    echo "   Builds the Dockerfile;"
+    echo "   Runs the docker image passing along the initial arguments."
+    echo
+    echo -e "${ORANGE}ARGUMENTS${NC}"
+    echo -e "${GREEN}   -s, --slug${NC}		Name of exercise slug"
+    echo -e "${GREEN}   -i, --input${NC}		Absolute path to solution folder (with trailing slash)"
+    echo -e "${GREEN}   -o, --output${NC}		Absolute path to output directory (with trailing slash)"
+    echo
+    echo -e "${ORANGE}OUTPUT${NC}"
+    echo "   Writes the test results to a results.json file in the passed-in output directory."
+    echo "   The test results are formatted according to the specifications at https://github.com/exercism/automated-tests/blob/master/docs/interface.md"
     exit 1
+}
+
+if [ $# -eq 0 ]; then
+    printHelpAndExit
 fi
 
+#Parse keys
+while(( "$#" )); do
+    case "$1" in
+        -s|--slug)
+            SLUG="$2"
+            shift
+            shift
+            ;;
+        -i|--input)
+            INPUT_DIR="$2"
+            shift
+            shift
+            ;;
+        -o|--output)
+            OUTPUT_DIR="$2"
+            shift
+            shift
+            ;;
+        *) # preserve positional arguments
+            PARAMS="$PARAMS $1"
+            shift
+            ;;
+    esac
+done
+
 # build docker image
-docker build --rm --no-cache -t swift-test-runner .
+docker build --rm --no-cache -t exercism/swift-test-runner .
```

Comment:
This is the actual name we use for the test runner, so adding the prefix ensure that the local development environment uses this built version.